src/Symfony/Component/Debug/Tests/DebugClassLoaderTest.php

```diff
@@ -275,11 +297,6 @@ public function loadClass($class)
     {
     }
 
-    public function getClassMap()
-    {
-        return array(__NAMESPACE__.'\Fixtures\NotPSR0bis' => __DIR__.'/Fixtures/notPsr0Bis.php');
-    }
```

Comment:
From what I saw this is not used and this class is already managed in `findFile` anyway.